packages/c/calamares/package.yml

```diff
@@ -17,36 +17,36 @@ summary    : Distribution-independent installer framework
 description: |
     Calamares is a distribution-independent system installer, with an advanced partitioning feature for both manual and automated partitioning operations. Calamares is designed to be customizable by distribution maintainers without the need for cumbersome patching, thanks to third-party branding and external modules support.
 builddeps  :
-    - pkgconfig(Qt5Core)
-    - pkgconfig(Qt5Quick)
-    - pkgconfig(Qt5Svg)
-    - pkgconfig(Qt5UiTools)
-    - pkgconfig(Qt5WebEngineWidgets)
+    - pkgconfig(Qt6Core)
+    - pkgconfig(Qt6Quick)
+    - pkgconfig(Qt6Svg)
+    - pkgconfig(Qt6UiTools)
+    - pkgconfig(Qt6WebEngineWidgets)
     - pkgconfig(icu-i18n)
     - pkgconfig(libparted)
-    - pkgconfig(polkit-qt5-1)
+    - pkgconfig(polkit-qt6-1)
     - pkgconfig(pwquality)
     - pkgconfig(python3)
     - pkgconfig(yaml-cpp)
     - extra-cmake-modules
```

Comment:
You won't need `extra-cmake-modules`, it's a transitive rundep of many of the `kf6-*-devel` packages